acm_web_site/apps/business/activities/serializers.py

```diff
@@ -27,6 +28,7 @@ class ActivitySerializer(serializers.ModelSerializer):
     class Meta:
         model = Activity
         fields = '__all__'
+        depth = 1
```

Comment:
what is this for?

Reply:
This determines how many levels of depth are searched in the database. For example in the relationships Activity - ActivityMember - Member tables, having depth = 1 when retrieving Activity information shows also the complete information of the members associated as shown below. 

![Captura de pantalla de 2020-07-29 17-01-52](https://user-images.githubusercontent.com/46963764/88858377-85adb780-d1bd-11ea-9106-8e3d532f2f7f.png)

When depth is not specified the same query returns the next results:
![Captura de pantalla de 2020-07-29 17-07-22](https://user-images.githubusercontent.com/46963764/88858689-02409600-d1be-11ea-9fcf-298c63301303.png)